packages/clerk-js/src/ui/common/NotificationCountBadge.tsx

```diff
@@ -1,16 +1,17 @@
 import { Flex, NotificationBadge } from '../customizables';
 import { useDelayedVisibility, usePrefersReducedMotion } from '../hooks';
-import type { ThemableCssProp } from '../styledSystem';
+import type { PropsOfComponent, ThemableCssProp } from '../styledSystem';
 import { animations } from '../styledSystem';
 
-type NotificationCountBadgeProps = {
-  notificationCount: number;
+type NotificationCountBadgeProps = PropsOfComponent<typeof NotificationBadge> & {
+  notificationCount: number | string;
   containerSx?: ThemableCssProp;
 };
 
-export const NotificationCountBadge = ({ notificationCount, containerSx }: NotificationCountBadgeProps) => {
+export const NotificationCountBadge = (props: NotificationCountBadgeProps) => {
+  const { notificationCount, containerSx, ...restProps } = props;
   const prefersReducedMotion = usePrefersReducedMotion();
-  const showNotification = useDelayedVisibility(notificationCount > 0, 350) || false;
+  const showNotification = useDelayedVisibility(!!notificationCount, 350) || false;
```

Comment:
🙃 feel free to ignore,
```suggestion
const showNotification = useDelayedVisibility(+notificationCount > 0, 350) || false;
```

Reply:
Thank you highlighting that, I've done some changes to bring this back this behaviour, you can take a look at the code again!